README.rst

```diff
@@ -303,6 +303,28 @@ The complete API is:
 All parameters have the same meaning as in the ``get_next_value`` and
 ``get_last_value`` functions.
 
+
+Examples
+========
+
+Reset values on a date
+----------------------
```

Comment:
I suggest we don't call this "resetting", since no sequence is being reset. Instead they're independent sequences.

```suggestion
Per-date sequences
------------------
```